includes/definitions.inc.php

```diff
@@ -520,6 +520,26 @@
 $config['os'][$os]['over'][0]['text']  = 'Device Traffic';
 $config['os'][$os]['over'][1]['graph'] = 'device_processor';
 $config['os'][$os]['over'][1]['text']  = 'CPU Usage';
+$config['os'][$os]['poller_modules']['aruba-controller']       = 0;
+$config['os'][$os]['poller_modules']['bgp-peers']              = 0;
```

Comment:
I considered that too, but didn't have any running BGP so I couldn't test, however looked at ftp://ftp.cisco.com/pub/mibs/supportlists/asa/asa-supportlist.html and can't see that polling BGP info should be supported(Same goes with OSPF).